### ISSUE TITLE
training / validation split

### ISSUE THREAD

**Issue by MartinHahner** [7dlabs/synscapes-utils#1]
Hi,
thanks for providing the dataset to us.

In your paper, you write under 6.1. Semantic segmentation:
"In order to explore statistically how DeepLab’s pre-trained Cityscapes model behaves, we run prediction on the Synscapes training set (24,000 images)"

Can you please provide the 1000 image IDs you excluded for those experiments?

**Comment by wyfeng1020** [comment]
I'm also curious about whether there is an official train/val/test split.

**Comment by mayone23** [comment]
Dear contributors,

I have downloaded all the compressed files and extracted it. But after extraction the segmentation labels and meta-data are missing. 
Could you please check on the same?

Thanks 
Ankit
